components/logins/README.md

```diff
@@ -67,6 +60,13 @@ mechanism). It can then create, read, update and delete login records from the d
 ### Examples
 - [Android integration](https://github.com/mozilla-mobile/android-components/blob/master/components/service/sync-logins/README.md)
 
+### Telemetry
+
+This component supports collecting quality and performance telemetry using the [Glean SDK](https://mozilla.github.io/glean/),
+but this collection is disabled by default. Applications that send telemetry via Glean may opt-in to gathering these metrics by
```

Comment:
This would be an exception to the way other components work, I think we really need a general solution to this problem, and I'm willing to prioritize this.